app/views/layouts/application.html.erb

```diff
@@ -9,6 +9,7 @@
   </head>
```

Comment:
Try `haml` and more better `hamlit` as it's faster
[source](https://github.com/k0kubun/hamlit)